database/mariadb_actions.yml

```
@@ -4,6 +4,9 @@ actions:
     - setSudoer: sqldb
     - log: "## Setup MariaDB logrotate on ${this}"
     - cmd[sqldb]: |-
+        mkdir /etc/systemd/system/mysql.service.d /etc/systemd/system/mariadb.service.d
+        echo -e "[Service]\nLimitNOFILE=524290" > /etc/systemd/system/mariadb.service.d/override.conf
+        echo -e "[Service]\nLimitNOFILE=524290" > /etc/systemd/system/mysql.service.d/override.conf
```

That's the good way to do it but I can't tell about the value set here.
What about the filesystem limit (fs.file-max) ?

```suggestion
        for n in mariadb mysql; do
          mkdir /etc/systemd/system/$n.service.d
          echo -e "[Service]\nLimitNOFILE=524290" > /etc/systemd/system/$n.service.d/override.conf
        done
```
or
```suggestion
        mkdir /etc/systemd/system/{mysql,mariadb}.service.d
        echo -e "[Service]\nLimitNOFILE=524290" > /etc/systemd/system/{mysql,mariadb}.service.d/override.conf
```

---

524290 is the value set in limits.conf file, and the actual value on slaves nodes during envs creation.
I fixed the (almost) duplicate line!